.github/workflows/release.yml

```diff
@@ -48,6 +48,9 @@ jobs:
       commit_message: 'Update examples link in index.html.markdown for ${{ github.event.inputs.version_number }} release'
       user_email: 'releasebot@mongodb.com'
       user_name: 'releasebot'
+    secrets:
+      GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
+      PASSPHRASE: ${{ secrets.PASSPHRASE }}     
```

Comment:
for the commits made in the release process I believe we also need to include the PAT to be able to push (by passing `remote` input), would it make sense to make it a required input?

Reply:
Done! Changed to required and removed unnecessary if where setting the remote